PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+When opening a pull request to submit a new HIP, please use the suggested template: (Link to HIP Template Github)
```

Comment:
Link is missing. Also we don't have a bot or a build process, so we can take those details out for now.